pkg/controller/vip.go

```diff
@@ -364,6 +411,66 @@ func (c *Controller) acquireIPAddress(subnetName, name, nicName string) (string,
 	}
 }
 
+func (c *Controller) handleUpdateVirtualParents(key string) error {
+	cachedVip, err := c.virtualIpsLister.Get(key)
+	if err != nil {
+		if k8serrors.IsNotFound(err) {
+			return nil
+		}
+		return err
+	}
+	// only pods in the same namespace as vip are allowed to use aap
+	if cachedVip.Status.V4ip == "" || cachedVip.Spec.Namespace == "" {
+		return nil
+	}
+	// add new virtual port if not exist
+	if err = c.OVNNbClient.CreateVirtualLogicalSwitchPort(cachedVip.Name, cachedVip.Spec.Subnet, cachedVip.Status.V4ip); err != nil {
```

Comment:
When this port will be deleted

Reply:
> When this port will be deleted

this port will be deleted when vip is deleted